lib/Importer/ONNXModelLoader.cpp

```diff
@@ -2606,6 +2606,30 @@ Error ONNXModelLoader::loadBatchNormalization(
   return Error::success();
 }
 
+Error ONNXModelLoader::loadInstanceNormalization(
+    const ONNX_NAMESPACE::NodeProto &op, ArgumentDictionaryTy &dict) {
+  const std::string &opName = loadOperatorName(op);
+
+  NodeValue in;
+  ASSIGN_VALUE_OR_RETURN_ERR(in, getNodeValueByName(op.input(0)));
+  Constant *scale;
```

Comment:
Is there a reason we use `Constant` rather than `NodeValue` here? I feel like we can possibly pass an output of a node as scale or bias.

Reply:
Was trying to stay consistent with similar nodes (e.g., BathcNormalization), but they have been recently modified to use NodeVlaue too. So updating InstanceNormalization as you suggested.